<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="TextEditLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>761</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QCheckBox" name="CheckBoxQMLSplashScreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show as Splash</string>
     </property>
    </widget>
    <widget class="QFrame" name="FrameSpinner">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditQML">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>spin.qml</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonQMLShow">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>4</number>
     </property>
     <property name="text">
      <string>QtQuick widget file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonQMLBrowseFile">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Browse for file">Select File</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>QQuickWidget Tester</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonUdpServer">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Udp Server</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>DPath</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>531</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QLineEdit" name="EditWatcher">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>411</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonWatcherAlive">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>IsAlive</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonCheck">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonStop">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonPathWatcher">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonStart">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonWatcherBrowseFile">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string extracomment="Browse for file">Select File</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonWatcherBrowseDir">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string extracomment="Browse for dir">Select Dir</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File or folder to watch</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse File</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse Dir</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>DPathWatcher</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>541</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="TextEditExtList">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>104</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="TextEditNameList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>131</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDPathBrowseDir">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string extracomment="Browse for dir">Select Dir</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="EditDPath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>411</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDPathSearch">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Count</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CheckBoxDPathRecoursive">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Recoursive</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CheckBoxDPathCase">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Case Sensistive</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RadioButtonDPathFiles">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>130</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Find Files</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="RadioButtonDPathDirs">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Find Dirs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Source Folder</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name patterns</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ext patterns</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse Dir</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Find files and folders</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>DShell</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="EditDShell">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>271</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDShellExecute">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDShellBrowseFile">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string extracomment="Browse for dir">Select File</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="TextEditDShellArgs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Args List</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CheckBoxExecuteDetached">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Detached (Wait unused)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="SpinBoxWaitMillis">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wait Millis</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="LabelDShellExecute">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Execute process</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1448</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
